django_project/feti/models/course.py

```diff
@@ -21,6 +23,7 @@
 class Course(models.Model):
 
     id = models.AutoField(primary_key=True)
+    campuses = models.ManyToManyField(Campus)
```

Comment:
I think it makes more sense to think of Campuses as having courses than as courses having campuses.